### DISCUSSION TITLE
(Mix) Could not compile dependency :credentials_obfuscation, "escript.exe

### DISCUSSION THREAD

**Discussion by SyedMuzamilM** [rabbitmq/credentials-obfuscation#12] [General]
While compiling my project. I am getting this error:

`
(Mix) Could not compile dependency :credentials_obfuscation, "escript.exe "c:/Users/Syed Muzamil/.mix/rebar3" bare compile --paths="f:/projects_startup/dogehouse/kousa/_build/dev/lib/*/ebin"" command failed. You can recompile this dependency with "mix deps.compile credentials_obfuscation", update it with "mix deps.update credentials_obfuscation" or clean it with "mix deps.clean credentials_obfuscation"
`

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
I cannot comment with this much information but very likely you are not running a compatible Erlang version. This project requires Erlang/OTP 22.3 or later.

**Reply by SyedMuzamilM** [reply]
Yes. I was using Erlang/OTP 21. And that was causing the issue. Thanks.
